lib/flight_job/models/script.rb

```diff
@@ -154,6 +164,10 @@ def script_path
       end
     end
 
+    def notes_path
+      @notes_path ||= File.join(FlightJob.config.scripts_dir, id, 'notes.txt')
```

Comment:
Let's change this to `notes.md`.  The webapp is going to assume the notes are valid markdown and naming the file that way will result in nice syntax highlighting in `$EDITOR`.

Reply:
c2968f93078c9098653107f48c025a1bf2112d29